packages/tinacms/src/toolkit/fields/plugins/mdx-field-plugin/plate/components/plate-ui/templates-toolbar-button.tsx

```diff
@@ -25,7 +25,7 @@ interface EmbedButtonProps {
 }
 
 const EmbedButton: React.FC<EmbedButtonProps> = ({ editor, templates }) => {
-  const [open, setOpen] = useState(false);
+  const openState = useOpenState();
```

Comment:
Let's just destructure here for brevity (so we don't need to call `openState.open` later on).

```suggestion
  const {open, onOpenChange} = useOpenState();
```